src/components/sections/Hero/Hero.tsx

```diff
@@ -69,6 +69,13 @@ const Hero = ({
           options={{
             fitIn: true,
           }}
+          // reset gatsby image default style
+          style={{
+            overflow: undefined,
+            position: undefined,
+          }}
+          // for mobile load 100vw image, for desktop load half img
+          sizes="(max-width: 768px) 100vw, 53vw"
```

Comment:
We have similar approach on [ProductCard](https://github.com/vtex-sites/base.store/blob/dd8df7195b5191e96f68900d3db1b8ad806245cc/src/components/product/ProductCard/ProductCard.tsx#L31). 

Maybe `50vw`, instead of `53vw`?

Reply:
The size of the current image is 53% of the whole screen

Reply:
53vw will get an image with a higher resolution, that works for this 53% image width and for PDP that uses 50%. Does it make sense?